src/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__image {
+  display: block;
+
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--c-gray);
+
+  margin-bottom: 16px;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.product-card__rewievs {
+  display: flex;
+  align-items: center;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.product-card__text {
+  font-weight: 400;
+  color: var(--c-gray);
+}
+
+.product-card__number {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: var(--c-darkBlue);
+
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  background: var(--c-blue);
+}
+
+.product-card__buy:hover {
+  background: var(--c-light);
+  border: var(--c-blue);
+  color: var(--c-blue);
+}
```

Comment:
this is how hover looks
![image](https://user-images.githubusercontent.com/37161315/216446911-5f605027-439c-4207-a8c1-852c4ecd57d0.png)